src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -980,6 +980,11 @@ protected TypeDesc NativeElementType
             }
         }
 
+        protected override void SetupArgumentsForFieldMarshalling()
```

Comment:
Another left-over?

https://github.com/dotnet/runtimelab/pull/1102/#discussion_r638519817

Reply:
I create small test and returning `S_INTArray_Seq` lead to 
```
--------------------------------------------------
Debug Assertion Violation

Message: Hardware exception raised inside the runtime.

Expression: 'ASSERT_UNCONDITIONALLY'

File: D:\d\kant\GitHub\dotnet\runtimelab\src\coreclr\nativeaot\Runtime\EHHelpers.cpp, Line: 560
--------------------------------------------------
```
with or without `SetupArgumentsForReturnValueMarshalling`, so that change definitely not helpful.

Reply:
Do you plan to revert this part then?

Reply:
I may confuse you. This line which we discuss are useful. Other one which I remove not.